test/powershell/Language/Scripting/Dynamicparameters.Tests.ps1

```diff
@@ -0,0 +1,68 @@
+﻿Describe "Dynamic parameter support in script cmdlets." -Tags "CI" {
+    BeforeAll {
```

Comment:
added

---
In reply to: [76886461](https://github.com/Powershell/Powershell/pull/2118#discussion_r76886461) [](ancestors = 76886461)